test/test-network/conf/25-qdisc-cake.network.d/bandwith.conf

```diff
@@ -0,0 +1,4 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+[CAKE]
+Bandwidth=250M
```

Comment:
```
/run/systemd/network/25-qdisc-cake.network: More than one root qdisc section is defined. Ignoring the qdisc section from line 9.
```

qdisc sections like `[CAKE]` does not support override.
Please directly edit .network file something like the following:
```python
with open(os.path.join(network_unit_dir, '25-qdisc-cake.network'), mode='w', encoding='utf-8') as f:
        f.write('Bandwidth=250M\n')
```